others/playfair_cipher.cpp

```diff
@@ -0,0 +1,156 @@
+//code for playfair cipher in cryptography
+
```

Comment:
/**
 * This is the code for playfair cipher
 * @author Ananya (https://github.com/Ananya322)
 */